intranet/apps/polls/models.py

```diff
@@ -94,6 +94,23 @@ def get_users_voted(self):
                 users = list(q.get_users_voted())
         return users
 
+    def get_num_eligible_voters(self):
+        users = []
```

Comment:
This method makes highly inefficient queries against the database by needlessly loading possibly large querysets into memory. I do acknowledge other methods in this file (particularly `get_user_voted()`) do not serve as a good example.
I would reference the Django [docs on queries](https://docs.djangoproject.com/en/3.0/topics/db/queries/) and the Django [docs on querysets](https://docs.djangoproject.com/en/3.0/ref/models/querysets/) which are good resources when trying to construct nice and efficient DB queries